src/FlyWithLua.cpp

```diff
@@ -1015,6 +1018,8 @@ void send_delayed_quarantined_message();
 
 int found_bad_function_script = 0;
 
+int developer_mode = 0;
```

Comment:
Now we have two flags related to developer mode: this and DevMode. Should we leave just one?

Reply:
I did the best I could without breaking it.